src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models.mdx

```diff
@@ -22,8 +22,8 @@ import accountsBillingUI from 'images/accounts_screenshot-crop_billing-UI.webp'
 
 New Relic organizations are on one of two pricing models, and we have separate sets of docs for each pricing model:
 
-* Our primary usage-based pricing ([see docs](/docs/new-relic-one-pricing-billing)): this is our newer pricing model, which bills based on usage: data ingested and number of billable users. Almost all New Relic organizations are on this model. All organizations created on or after July 30, 2020 are on this model, as are some older organizations that have [switched to this model](/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models/). 
-* Our original product-based pricing model ([see docs](/docs/accounts/original-accounts-billing/product-pricing/product-based-pricing)): our older model involves subscriptions to specific products. 
+* Our primary usage-based pricing ([see docs](/docs/new-relic-one-pricing-billing)): This is our newer pricing model, which bills based on usage: data ingested and number of billable users. Almost all New Relic organizations are on this model. All organizations created on or after July 30, 2020 are on this model, as are some older organizations that have [switched to this model](/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models/). 
```

Comment:
```suggestion
* Our primary usage-based pricing ([see docs](/docs/new-relic-one-pricing-billing)): This is our newer pricing model, which bills based on data ingested and number of billable users. Almost all New Relic organizations are on this model. All organizations created on or after July 30, 2020 are on this model, as are some older organizations that have [switched to this model](/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models/). 
```

Reply:
@akristen your removal of "usage" might help reduce confusion in the future. Let's go with that.